database/migrations/2019_02_05_083929_add_primary_key_to_submissions.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use App\Submission;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Ramsey\Uuid\Uuid;
+
+class AddPrimaryKeyToSubmissions extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('submissions', function (Blueprint $table) {
+            $table->uuid('id')->default(0);;
+            $table->primary('id');
+        });
+
+        Submission::all()->map(function ($submission){
+            $submission->id = Uuid::uuid4();
+            $submission->save();
+        });
```

Comment:
Should we modify it to not have a default after this?

Reply:
:+1: